src/DreamCommerce/Component/BugTracker/CollectorExtension/CollectorExtensionPriorityQueue.php

```diff
@@ -0,0 +1,156 @@
+<?php
+namespace DreamCommerce\Component\BugTracker\CollectorExtension;
+
+use SplPriorityQueue;
+
+class CollectorExtensionPriorityQueue extends SplPriorityQueue implements CollectorExtensionQueueInterface
+{
+    const   NAME_KEY = 'name',
```

Comment:
Tak na przyszłość stałe zawsze wrzucaj do interfejsów. Co do tej klasy to nie za bardzo podoba mi się powielanie funkcjonalności które dają te klasy:

https://github.com/zendframework/zend-stdlib/blob/master/src/SplPriorityQueue.php
https://github.com/dreamcommerce/bugtracker-bundle/blob/master/src/DreamCommerce/Component/BugTracker/Collector/CollectorPriorityQueue.php

Trzeba dorzucić klasę która będzie w stanie rejestrować tylko klasy dziedziczące po jakimś interfejsie w common-bundle. Zajmę się tym